### PR TITLE
Google Vision API support

### PR DESCRIPTION
**Pull request by f1nzer**
Google Vision API can be enabled via UseNativeScanning option.

### CODE REVIEW

File: Source/ZXing.Net.Mobile.Android/ZXing.Net.Mobile.Android.csproj [hunk 1]
```diff
@@ -12,7 +12,10 @@
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <RootNamespace>ZXing.Mobile</RootNamespace>
     <AssemblyName>ZXingNetMobile</AssemblyName>
-    <TargetFrameworkVersion>v4.0.3</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
```
Comment (F1nZeR) [inline Source/ZXing.Net.Mobile.Android/ZXing.Net.Mobile.Android.csproj]: Google Play Service SDK's requires a Target Framework of at least Android 4.1 (API Level 16) to compile.
Is it ok to set latest paltform sdk or 4.1 is more preferable?

Comment (Redth) [inline Source/ZXing.Net.Mobile.Android/ZXing.Net.Mobile.Android.csproj]: This is an interesting topic.  We really should have TargetFrameworkVersion match the nuspec file's target framework version.  I hesitate to set this to 6.0 at this point just due to backwards compatibility reasons, can we bump it to 4.1 for now and also change the nuspec files to match?

**Comment by Redth** [comment]
I love this idea, it was definitely on my list of things I wanted to add, however, I'm not sure if forcing a dependency on Google Play Services is the right thing to do.  Can we refactor this into its own assembly and nuget package so that users can optionally pull it down to use it?

**Comment by Redth** [comment]
Can you rebase this on master please?

**Comment by ztl8702** [comment]
Interesting. Would love to see this moving forward. The code doesn't seem too complicated, I will take another look and see what I can do here.

**Pull request closed without merging**